### PR TITLE
Deploy contracts to mainnet

### PR DESCRIPTION
**Pull request by Lbqds**

### CODE REVIEW

File: clients/js/evm.ts [hunk 1]
```diff
@@ -480,3 +486,77 @@ export async function setStorageAt(rpc: string, contractAddress: string, storage
...
+  provider: ethers.providers.JsonRpcProvider
+) {
+  const tokenBridge = ethers_contracts.BridgeImplementation__factory.connect(tokenBridgeAddress, provider)
+  checkChainParams(
+    (await tokenBridge.chainId()) as ChainId,
+    await tokenBridge.governanceChainId(),
+    await tokenBridge.governanceContract(),
+    expectedChainId
+  )
+  const wrappedNative = await tokenBridge.WETH()
```
Comment (h0ngcha0) [inline clients/js/evm.ts:541]: Q: Dose `WETH` return `WBNB` for BSC chain?

Comment (Lbqds) [reply clients/js/evm.ts:541]: Yes, in the wormhole contract, `WETH` is used to represent the wrapped native address on EVM chains.

**Review by h0ngcha0 — APPROVED** [review]
LGTM :+1:

**Review by polarker — APPROVED** [review]
LGTM

**Pull request merged**